src/erlcloud_as.erl

```diff
@@ -78,6 +82,20 @@
 -define(COMPLETE_LIFECYCLE_ACTION_ACTIVITY, 
         "/CompleteLifecycleActionResponse/ResponseMetadata/RequestId").
 
+-define(COMPLETE_LIFECYCLE_ACTION_RESPONSE,
+    "/CompleteLifecycleActionResponse/ResponseMetadata/RequestId").
+
+-define(RECORD_LIFECYCLE_ACTION_HEARTBEAT_RESPONSE,
+    "/RecordLifecycleActionHeartbeatResponse/ResponseMetadata/RequestId").
+
+
+%% --------------------------------------------------------------------
+%% @doc Sets as hosts in #aws_config{}
+%% @end
+%% --------------------------------------------------------------------
+set_as_host(Host) ->
```

Comment:
should not this come by magic from erlcloud_aws with service config?
unless you set this for "MyLocalASHost"...